artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/etc/commented-ping-settings.txt

```diff
@@ -1,6 +1,6 @@
       <!--
-        You can specify the NIC you want to use to verify if the network
-         <network-check-NIC>theNickName</network-check-NIC>
+        You can verify the network health of a particular NIC by specifying the <network-check-NIC> element.
```

Comment:
I don't think you can use <> inside comments...

Reply:
According to https://www.w3.org/TR/REC-xml/#sec-comments, everything except double hyphen should be fine.

Reply:
I used some online XML validator and it says it is ok.

Reply:
my bad.. I thought you couldn't 👍

Reply:
In fact.. I used it before.. duh!